app/controllers/admin/articles_controller.rb

```diff
@@ -26,6 +28,7 @@ def new
   # /admin/articles/1/edit
   def edit
     @collection = Article.find(@article.collection_id) if @article.in_collection?
+    @title = t('.edit_title', id: @article.id, title: @article.title, subtitle: @article.subtitle)
```

Comment:
actually, I have an idea to make it work. I will add a commit on top of this and you can see which you prefer